library/core/src/ptr/non_null.rs

```diff
@@ -1770,3 +1779,30 @@ impl<T: ?Sized> From<&T> for NonNull<T> {
         unsafe { NonNull { pointer: reference as *const T } }
     }
 }
+
+//#[unstable(feature="kani", issue="none")]
+mod verify {
+    use super::*;
+
+    // pub const unsafe fn new_unchecked(ptr: *mut T) -> Self
+    #[kani::proof_for_contract(NonNull::new_unchecked)]
+    pub fn non_null_check_new_unchecked() {
+        let mut x : i32 = kani::any();
```

Comment:
```suggestion
        let mut x: i32 = kani::any();
```
You can run `cargo fmt` to automatically fix those code-formatting issues.

Reply:
I see. Perhaps try running the `rustfmt` executable directly on the file, e.g. `rustfmt non_null.rs`.